app/services/katello/pulp3/ansible_collection.rb

```diff
@@ -4,7 +4,7 @@ class AnsibleCollection < PulpContentUnit
       include LazyAccessor
 
       def self.content_api
-        PulpAnsibleClient::ContentCollectionsApi.new(Katello::Pulp3::Repository::AnsibleCollection.api_client(SmartProxy.pulp_master!))
+        PulpAnsibleClient::ContentCollectionVersionsApi.new(Katello::Pulp3::Repository::AnsibleCollection.api_client(SmartProxy.pulp_master!))
       end
```

Comment:
This is handle renaming of ContentCollectionsApi to ContentCollectionVersionApi.
Those are the 2 changes as part of the gem updates I found.